pulsar-broker/src/main/java/org/apache/pulsar/broker/service/AbstractBaseDispatcher.java

```diff
@@ -220,6 +220,12 @@ public int filterEntriesForConsumer(Optional<EntryWrapper[]> entryWrapper, int e
         if (CollectionUtils.isNotEmpty(entriesToFiltered)) {
             subscription.acknowledgeMessage(entriesToFiltered, AckType.Individual,
                     Collections.emptyMap());
+
+            int filtered = entriesToFiltered.size();
```

Comment:
It's actually entry count, not message count. Please make it consist.

Reply:
make sense, I'll change the field name. Thanks!